test/ctia/http/generative/fulltext_search_test.clj

```diff
@@ -319,65 +319,68 @@
                       gen/generate)
             ignore-ks  [:created :groups :id :incident_time :modified :owner :timestamp]]
 
-        ;; to avoid accidentally picking up previous state
-        (store/delete-search incidents-store
```

Comment:
already cleaned with the `for-each-es-version` macro and `#(es-index/delete! % "ctia_*")`